mnefun/_mnefun.py

```diff
@@ -801,19 +844,19 @@ def run_sss_command(fname_in, options, fname_out, host='kasga', port=22,
             pass
 
 
-def run_sss_positions(fname_in, fname_out, host='kasga', opts='', port=22):
+def run_sss_positions(p, fname_in, fname_out, opts=''):
```

Comment:
Don't change the signature of these `run_` functions. They're designed to be used outside of `analysis` scripts, as standalone calls.